deno_webgpu/error.rs

```diff
@@ -182,6 +183,17 @@ impl Display for GPUError {
 
 impl std::error::Error for GPUError {}
 
+impl GPUError {
+    fn from_webgpu(e: impl WebGpuError) -> Self {
+        match e.webgpu_error_type() {
+            ErrorType::Internal => GPUError::Internal,
+            ErrorType::DeviceLost { reason } => GPUError::Lost(reason.into()),
```

Comment:
`GPUError::Lost` is marked with `#[class("UNREACHABLE")]` but this mapping looks consistent with the previous one:

```
DeviceError::Lost => GPUError::Lost(GPUDeviceLostReason::Unknown),
```

@crowlKats deno should probably ignore the lost variant since the lost callback on the device will be called when the device became lost. We could then remove the inner `reason` field.

Reply:
I will consider this resolved, but leave this open for visibility. Ignoring `DeviceLost` and removing the `reason` field SGTM; does that mean the `GPUError::Lost` variant should be removed, @crowlKats?